pkg/download/goget/goget.go

```diff
@@ -125,7 +125,7 @@ func (gg *goget) list(op errors.Op, mod string) (*listResp, error) {
 }
 
 func (gg *goget) GoMod(ctx context.Context, mod string, ver string) ([]byte, error) {
-	const op errors.Op = "goget.Info"
+	const op errors.Op = "goget.GoMod"
```

Comment:
Please add package name also. for example download.goget.GoMod
There is a WIP PR that does that, but I am happy to rebase it on top of your changes.

https://github.com/gomods/athens/pull/604/files

Reply:
@manugupt1 can we do this and the below change in a follow-up? or, if it's not too bad for you to rebase, is that ok?

Reply:
Yes, GoGet has been removed so we do not need this change anymore.
Resolving merge conflicts  should fix this automatically

Reply:
Just FYI we don't need to make this *sound* like a stack trace with exact package-file-method operations.

Since it's just a string we can make it more human readable and easy to look up and understand when debugging in cloud logs.

In other words this can be `downloadProtocol.GoMod` or `gogetProtocol.GoMod` -- or simply just `goget.GoMod` because those strings are likely unique and easy to scan in your eye but also easy to do global look ups when you're searching the code base 

So I think we should keep it short and sweet, but also understandable.

Reply:
Yep! I do not really want it to sound like a stack trace. The reason I like this has got less to do with the length but for the following reasons:

1) It has a convention, so it is easy to ask others to stick to it
2) It is easier to filter logs, so if you want to filter logs you can separate by the delimiter and check it out with a simple regex.

Reply:
Oh also! I am saying something like package.object.func rather than package.file.func

Reply:
@manugupt1 i think the convention is difficult to figure out, not always do you have something like `package.object.func` sometimes you just have `package.func` or `package.globalVar` etc. 

The nice thing about function operations is that it's meant for humans, yet it can still be filterable. So keeping things short and understandable like `goget.GoMod` is both easier for humans to read and easier for humans to filter. 

Most likely, you're gonna read an array of operations so reading some thing like 
```["download.GoMod", "stasher.Pool", "goget.GoMod", "mongo.Get"]``` 
is much easier on the eye than : 

```["download.Handlers.GoMod", "stasher.pool.Pool", "download.goget.GoMod", "storage.Mongo.Get"]```